specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/examples/LabelHistory_Get.json

```
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "containerAppName": "labelsapp",
```

nit: can ignore. To read label properties separately

```suggestion
    "containerAppName": "testContainerApp",
```

---

good point, for readability i fixed it